### PR TITLE
Update operator installation instructions to new prefix

### PR DESCRIPTION
**Pull request by masayag**
The new namespace and operator prefix is `sonataflow`. Therefore, the instructions are updated accordingly, and some of the outputs of the commands.

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

**Review by spolti — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 1]
```diff
@@ -57,26 +57,24 @@ You can follow the deployment of the {product_name} Operator:
...
 
 .Successful Installation Output
 [source]
 ----
-NAME                                                            READY   STATUS              RESTARTS   AGE
-kogito-serverless-operator-controller-manager-948547ffd-sr2j2   0/2     ContainerCreating   0          6s
-kogito-serverless-operator-controller-manager-948547ffd-sr2j2   1/2     Running             0          7s
-kogito-serverless-operator-controller-manager-948547ffd-sr2j2   2/2     Running             0          20s
+NAME                                                     READY   STATUS              RESTARTS   AGE
+sonataflow-operator-controller-manager-948547ffd-sr2j2   0/2     ContainerCreating   0          6s
```
[the remark this answers is not in the document]
Comment (masayag) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:69]: Restored the previous records.

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Thank you!

**Review by kaldesai — APPROVED** [review]
@masayag Your content looks good to me. Thank you!

**Review by MarianMacik — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**